godtools/Views/TractElements/Cards.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Cards.swift
+//  godtools
+//
+//  Created by Devserker on 5/2/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+//  NOTES ABOUT THE COMPONENT
+//  * Cards is a component that is not present on the XML. The component was generate
+//  because of the need to have a container to store all the cards. The cards container
+//  will only store elements of the kind Card.
+//  * Following the XML structure, the Cards container will always be a children of
+//  TractRoot container. Also, the Cards container will always be at the same level of a
+//  Header component.
+//  * The height size of this component will always be the same of TractRoot.height
+
+import Foundation
+import UIKit
+import SWXMLHash
+
+class Cards: BaseTractElement {
+    
+    enum CardsState {
+        case open, preview
+    }
+    
+    var cardsState = CardsState.preview
+    
+    var xPosition: CGFloat {
```

Comment:
Here's another example where 0.0 is returned from the getter. Can this be changed to just initialize it to 0.0, or can it be a let xPosition = 0.0 if it doesn't change?